scripts/troubleshoot/LogCollection/AKSInsightsLogCollection.sh

```diff
@@ -0,0 +1,226 @@
+#!/bin/bash
+#
+# Copyright (c) Microsoft Corporation.
```

Comment:
change the name of file from "AKSInsightsLogCollection.sh" to "AgentLogCollection.sh"

Reply:
changed the file name